chia/consensus/default_constants.py

```diff
@@ -72,7 +72,7 @@
     MAX_GENERATOR_SIZE=uint32(1000000),
     MAX_GENERATOR_REF_LIST_SIZE=uint32(512),  # Number of references allowed in the block generator ref list
     POOL_SUB_SLOT_ITERS=uint64(37600000000),  # iters limit * NUM_SPS
-    SOFT_FORK5_HEIGHT=uint32(5940000),
+    SOFT_FORK6_HEIGHT=uint32(6815000),
```

Comment:
If we want 124.8 days at the time of this comment, then this is correct.

Reply:
maybe that's overkill. I was thinking of a release on the 18th then + 30 days and round up. I see now cameron suggested a release on the 12th

Reply:
If I change it to 6800000 it's ~2025-03-22 which is closer to 2024-12-12 + 90

Reply:
Resolving this conversation due to approval of the PR and changes.